events.json

```diff
@@ -38,7 +54,7 @@
       },
       {
         "title": "Meetup",
-        "url": "http://www.meetup.com/Hackergarten-Basel/events/220246661/"
+        "url": "http://www.meetup.com/Hackergarten-Basel/events/220947562/"
```

Comment:
Wrong link to Hackergarten February '15 replaced by correct link to Hackergarten March '15